debian/control

```diff
@@ -2,28 +2,29 @@ Source: nfd
 Section: net
 Priority: optional
 Maintainer: Niv Sardi <xaiki@endlessm.com>
-Build-Depends: debhelper (>= 9),
-               dh-waf,
+Build-Depends: adduser,
```

Comment:
Just like in ndn-cxx, I don't think this should be in `Build-Depends`. `adduser` is called from `nfd.postinst`, so it should be in `Depends` there like libcap2-bin for setcap.